package.json

```diff
@@ -6,18 +6,18 @@
     "preinstall": "npx only-allow pnpm",
     "start": "docusaurus start",
     "build": "docusaurus build",
+    "serve": "docusaurus serve",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "release": "docusaurus docs:version"
   },
   "dependencies": {
-    "@docusaurus/core": "3.2.1",
-    "@docusaurus/preset-classic": "3.2.1",
-    "@docusaurus/theme-search-algolia": "3.2.1",
+    "@docusaurus/core": "3.5.2",
+    "@docusaurus/preset-classic": "3.5.2",
+    "@docusaurus/theme-search-algolia": "3.5.2",
     "@mdx-js/react": "^3.0.1",
-    "axios": "1.6.8",
-    "clsx": "^2.1.0",
-    "node": "^18.20.1",
```

Comment:
Add this back in as well. This ensures that the build is reporducible.

Reply:
Done.